internals-js/src/index.ts

```diff
@@ -7,11 +7,11 @@ export * from './types';
 export * from './operations';
 export * from './utils';
 export * from './debug';
-export * from './coreSpec';
-export * from './joinSpec';
-export * from './tagSpec';
-export * from './inaccessibleSpec';
-export * from './federationSpec';
+export * from './specs/coreSpec';
```

Comment:
should we sort those imports in some way?

Reply:
I don't disagree. We should add the eslint rule and probably in a separate PR. But until it's enforced / automated there's not much point to worrying about it here.